### PR TITLE
[gslice.array.comp.assign] Fix missing title

### PR DESCRIPTION
**Pull request by FrankHB**

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -8228,7 +8228,7 @@
 refers.
 \end{itemdescr}
 
-\rSec3[gslice.array.comp.assign]{\tcode{gslice_array}}
+\rSec3[gslice.array.comp.assign]{\tcode{gslice_array} computed assignment}
```
Comment (zygoloid) [inline source/numerics.tex]: This should presumably be "compound assignment, not "computed assignment", per [expr.ass].

Comment (FrankHB) [inline source/numerics.tex]: @zygoloid I'm not sure if it is correct. "Computed assignments" occurs in many other clauses, i.e. [valarray.syn]/5, [template.valarray.overview], [valarray.cassign], [slice.arr.comp.assign], [mask.array.comp.assign] and [indirect.array.comp.assign]. Is this editorial? Also not sure difference between "cassign" and "comp.assign" is intended.

**Comment by zygoloid** [comment]
I suspect someone got this wrong once a long time ago, and then the error was copied into other places.

**Comment by FrankHB** [comment]
Is it OK to replace all these occurrence to "compound"?

**Comment by zygoloid** [comment]
I expect so. If you put together a patch to do that, I'll go through them all and make sure none of them really did mean something else, but I suspect they're all incorrect.

**Comment by FrankHB** [comment]
Done.

**Comment by tkoeppe** [comment]
@FrankHB: Squash, rebase and repush?

**Comment by FrankHB** [comment]
OK.

**Comment by tkoeppe** [comment]
@zygoloid: PTAL

File: source/numerics.tex [hunk 2]
```diff
@@ -7280,11 +7280,11 @@
 If the array and the
 argument array do not have the same length, the behavior is undefined.%
 \indextext{undefined}
-The appearance of an array on the left-hand side of a computed assignment
+The appearance of an array on the left-hand side of a compound assignment
 does \tcode{not} invalidate references or pointers.
 
 \pnum
-If the value of an element in the left-hand side of a valarray computed
+If the value of an element in the left-hand side of a valarray coumpound
```
Comment (tkoeppe) [inline source/numerics.tex]: Check spelling

**Comment by tkoeppe** [comment]
Maybe make the commit message something slimmer that's just one line?

**Comment by FrankHB** [comment]
Sorry for the type. Fixed.

**Comment by tkoeppe** [comment]
You mean "typo"? :-)

**Pull request merged**